### ISSUE TITLE
[Project] Every scanner can be a super-scanner

### ISSUE THREAD

**Issue by matteomancini** [connecthon/2022#2]
**Title**: Every scanner can be a super-scanner
**Project Leader**: Carolyn McNabb (McNabbC1@cardiff.ac.uk)

**Description**: The goal of this project is to develop computational methods to infer Connectom-like diffusion-weighted MRI data at ultra-strong gradients from data collected at weaker gradients on a clincal 3T Siemens Prisma scanner. How? Two words: deep learning!

**Ideal participant requirements**: experience with the Python programming language and the PyTorch deep-learning package.

**Resources**: WAND data, local installation of a recent version of the Python interpreter as well as the PyTorch and TorchIO packages, GPUs from online services (colab/paperspace/kaggle) as well as on local workstations.

**Good first issues**:

1. Predict high b-values from low b-values: 
- Training a UNet-like architecture [1,2];
- Training a GAN-like architecture (e.g. Pix2Pix [3]);
2. Compare the quality of the prediction against ground-truth (e.g. MSE);
3. Assess the networks' performances in pathology (e.g. in patients with lesions or atrophy).

**References**

[1] [https://www.sciencedirect.com/science/article/pii/S1053811920305036](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.sciencedirect.com%2Fscience%2Farticle%2Fpii%2FS1053811920305036&data=05%7C01%7CPalomboM%40cardiff.ac.uk%7C26414c7cb7d1467dab5f08da2945163e%7Cbdb74b3095684856bdbf06759778fcbc%7C1%7C0%7C637867673934705966%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=UvzyHqz2Fr%2FPH80UBhBuZ5Pt%2BvqbHOYyFP%2BN8hEyXAw%3D&reserved=0)

[2] [https://github.com/qiyuantian/DeepDTI](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fqiyuantian%2FDeepDTI&data=05%7C01%7CPalomboM%40cardiff.ac.uk%7C26414c7cb7d1467dab5f08da2945163e%7Cbdb74b3095684856bdbf06759778fcbc%7C1%7C0%7C637867673934705966%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=mDMExA6cV1W%2BIAkpjT6uGnH3dBK5fvwNNNxqc3np%2BfA%3D&reserved=0)

[3] [https://www.tensorflow.org/tutorials/generative/pix2pix](https://www.tensorflow.org/tutorials/generative/pix2pix)
